test/extensions/filters/http/ext_proc/utils.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "envoy/config/core/v3/base.pb.h"
+
+#include "absl/strings/string_view.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+extern void expectHttpHeader(const envoy::config::core::v3::HeaderMap& headers,
```

Comment:
Should this be some moral equivalent of `headerMapEqualIgnoreOrder`?

Reply:
Yep. I improved it in the last commit to just be a wrapper for that.